### ISSUE TITLE
Add a way to open a script from the script documentation page.

### ISSUE THREAD

**Issue by pcvonz** [godotengine/godot-proposals#11425]
### Describe the project you are working on

Nothing special

### Describe the problem or limitation you are having in your project

Navigating to the source script from the documentation page. 

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

A link from the documentation that opens up the script in the editor.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

https://github.com/user-attachments/assets/881e4716-a4ec-4740-8539-c223671d91ec

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No, the generated documentation will need to be modified. 

### Is there a reason why this should be core and not an add-on in the asset library?

Requires a change to the editor

**Comment by arkology** [comment]
Looks like you already have implementation, right?

**Comment by pcvonz** [comment]
@arkology Yeah! Something pretty basic. I need to do a couple things though: 
- figure out how the documentation generation works and make sure the links aren't added
- Add links to the the source files for built-in classes (maybe that's a separate PR)
